dotnet/src/Functions/Functions.OpenApi/Model/RestApiOperationParameter.cs

```diff
@@ -85,6 +93,7 @@ public RestApiOperationParameter(
         string? arrayItemType = null,
         object? defaultValue = null,
         string? description = null,
+        string? format = null,
```

Comment:
This looks like a breaking change, but I assume we are ok with that in this case?

Reply:
This class, together with a few others, should have been marked as internal. As part of the OpenAPI graduation, their public API surface is going to be revised, and that change can be made then. Meanwhile, even though the class is public, the SK public API surface neither allows retrieving an instance of the class nor to accepting it. Therefore, the probability of someone creating an instance of the class is very low because there's no significant value in doing so.